packages/aws-rfdk/bin/stage-deadline.ts

```diff
@@ -179,10 +212,20 @@ The AWS CLI must be configured to authenticate using your AWS account. This can
 See https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html for documentation on how to configure the AWS CLI.
 
 Usage: stage-deadline [--output <output_dir>] [--verbose]
+                      [<deadline_release_version>]
```

Comment:
Since the version is currently a required argument for this calling pattern, we should remove the outer brackets with signify an optional argument.

Reply:
fixed.